release notes/v0.54.0.md

```diff
@@ -0,0 +1,304 @@
+
+Thank you @shiroyk for adding both the new array methods and BitInt :bow:.
+
+### New `setChecked` method for the browser module [`browser#1403`](https://github.com/grafana/xk6-browser/pull/1403)
+
+Previously, users could check or uncheck checkbox and radio button elements using the [`check`](https://grafana.com/docs/k6/latest/javascript-api/k6-browser/page/check/) and [`uncheck`](https://grafana.com/docs/k6/latest/javascript-api/k6-browser/page/uncheck/) methods. Now, we've added a `setChecked` method that allows users to set a checkbox or radio button to either the checked or unchecked state with a single method and a boolean argument.
+
+```javascript
+await page.setChecked('#checkbox', true);   // check the checkbox
+await page.setChecked('#checkbox', false);  // uncheck the checkbox
+```
+
+[Page](https://grafana.com/docs/k6/latest/javascript-api/k6-browser/page/setchecked/), [Frame](https://grafana.com/docs/k6/latest/javascript-api/k6-browser/frame/setchecked/), [ElementHandle](https://grafana.com/docs/k6/latest/javascript-api/k6-browser/elementhandle//setchecked/), and [Locator](https://grafana.com/docs/k6/latest/javascript-api/k6-browser/locator/setchecked/) now support the new `setChecked` method.
+
+### Async `check` function utility [`k6-utils#13`](https://github.com/grafana/k6-jslib-utils/pull/13)
+
+Writing concise code can be difficult when using [the k6 `check` function](https://grafana.com/docs/k6/latest/javascript-api/k6/check/) with async code since it doesn't support async APIs. A solution that we have suggested so far is to declare a temporary variable, wait for the value that is to be checked, and then check the result later. However, this approach can clutter the code with single-use declarations and unnecessary variable names, e.g.:
+
+```javascript
+const checked = await p.locator('.checked').isChecked();
+
+check(checked, {
+    'checked': c => c,
+});
+```
+
+To address this limitation, we've added a version of the `check` function to [jslib.k6.io](https://jslib.k6.io/) that makes working with `async`/`await` more ergonomic. The `check` function is a drop-in replacement for the built-in check, with added support for async code. Any `Promise`s will be awaited, and the result is reported once the operation has been completed:
+
+```javascript
+// Import the new check function from jslib.k6.io/k6-utils
+import { check } from 'https://jslib.k6.io/k6-utils/1.5.0/index.js';
+
+// ...
+
+// Use the new check function with async code
+check(page, {
+    'checked': async p => p.locator('.checked').isChecked(),
+});
+```
+
+Check out [the `check` utility function's documentation](https://grafana.com/docs/k6/latest/javascript-api/jslib/utils/check/) for more information on how to use it.
+
+### Support ArrayBufferViews in send for `k6/experimental/websockets` [#3944](https://github.com/grafana/k6/pull/3844)
```

Comment:
Does this require a docs update? 🤔

Reply:
already pushed https://github.com/grafana/k6-docs/commit/1f74a07536cbff40adf5fa7ff74df983e53a6646  ... as main turned out to not be protected from direct pushes. It is now

Reply:
Ah, I see it. Thanks @mstoykov. 🙇